MedTracker/Controllers/PatientController.cs

```diff
@@ -117,6 +117,31 @@ public async Task<IActionResult> GetTreatments()
 
     }
 
+    [HttpGet("GetDoctors")]
+    public async Task<IActionResult> GetDoctors()
+    {
+        try
+        {
+            // Fetch doctors and their information
+            var doctors = await _context.Doctors
+                .Select(d => new
+                {   
+                    DocID = d.IdUser,
```

Comment:
Please, use normal names.
Why first name is FName and phone number is PhoneNumber. If the intention was to shorten the name, it is not applied to all of them.
The names should be: DoctorId, FirstName, LastName, etc